charts/netdata/templates/clusterrole.yaml

```diff
@@ -11,11 +11,12 @@ metadata:
 rules:
   - apiGroups: [""]
     resources:
-      - "pods"        # used by sd, netdata (cgroup-name.sh, get-kubernetes-labels.sh)
-      - "services"    # used by sd
-      - "configmaps"  # used by sd
-      - "secrets"     # used by sd
-      - "nodes"       # used by go.d.plugin (k8s_state)
+      - "pods"           # used by sd, netdata (cgroup-name.sh, get-kubernetes-labels.sh)
+      - "services"       # used by sd
+      - "configmaps"     # used by sd
+      - "secrets"        # used by sd
+      - "nodes"          # used by go.d.plugin (k8s_state)
+      - "nodes/metrics"  # used by go.d.plugin (k8s_kubelet)
```

Comment:
Can this be covered by the following?

```yaml
  - nonResourceURLs: ["/metrics"]
    verbs:
      - "get"
```

Reply:
Not a specialist, but if yes, I think it is better - more explicit that we need a non-resource /metrics endpoint.

Reply:
It can't as nodes/metrics are actual resources.
I tested it anyway (my other wrong PR lowered my self-confidence ;-)) but it does not work

Reply:
Ah, you are right. Didn't know that, I guess I've found the relevant documentation - https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-authentication-authorization/#kubelet-authorization.

Let's then mention that it is when querying the HTTPS endpoint in the comment.

Reply:
Fixed in [fce91c7](https://github.com/netdata/helmchart/pull/276/commits/fce91c7de4d194eb28cfca6c9c4b6ef36fbf63cc)